### PR TITLE
Added method to create as-performed iri from as-planned iri

### PR DESCRIPTION
**Pull request by alwynmathew**

### CODE REVIEW

**Review by copyme — COMMENTED** [review]
please see the comment

File: DTP_API.py [hunk 1]
```diff
@@ -348,6 +353,24 @@ def query_all_pages(self, fetch_function, *fetch_function_arg):
 
         return query_response_all_pages
 
+    def create_as_performed_iri(self, as_planned_iri):
```
[the remark this answers is not in the document]
Comment (copyme) [reply DTP_API.py:356]: maybe we could have a class with rules for creating IRIs for different types? What do you think?

[the remark this answers is not in the document]
Comment (copyme) [reply DTP_API.py:356]: we can also for now keep it here and refactor later.

**Comment by alwynmathew** [comment]
@copyme Moved to helper for now

**Review by copyme — APPROVED** [review]

**Pull request merged**